.github/workflows/Format.yml

```
@@ -0,0 +1,43 @@
+name: Format Check
+on:
+  # because we rely on ReviewDog to trigger failure, there's no reason to run on anything
+  pull_request:
+    paths:
+      - "**/*.jl"
+      - ".github/workflows/FormatCheck.yml"
+concurrency:
+  # Skip intermediate builds: always.
+  # Cancel intermediate builds: only if it is a pull request build.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+jobs:
+  format-check:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        version:
+          - "1"
```

```suggestion
          - "1.9"
```

since you're using a manifest